common/djangoapps/enrollment/tests/test_data.py

```diff
@@ -257,3 +261,36 @@ def test_update_for_non_existent_user(self):
     def test_update_for_non_existent_course(self):
         enrollment = data.update_course_enrollment(self.user.username, "some/fake/course", is_active=False)
         self.assertIsNone(enrollment)
+
+    def test_get_course_with_expired_mode_included(self):
+        """ Verify that method returns expired modes if include_expired
+        is true. """
+        modes = ['honor', 'verified', 'audit']
+        self._create_course_modes(modes, course=self.course)
+        # Change verified mode expiration.
+        mode = CourseMode.objects.get(course_id=self.course.id, mode_slug=CourseMode.VERIFIED)
+        mode.expiration_datetime = datetime.datetime(year=1970, month=1, day=1, tzinfo=UTC)
+        mode.save()
+
+        result_course = data.get_course_enrollment_info(unicode(self.course.id), include_expired=True)
+        result_slugs = [mode['slug'] for mode in result_course['course_modes']]
+        for course_mode in modes:
+            self.assertIn(course_mode, result_slugs)
+
+    def test_get_course_without_expired_mode_included(self):
+        """ Verify that method does not returns expired modes if include_expired
+        is false. """
+
+        self._create_course_modes(['honor', 'verified', 'audit'], course=self.course)
+        # Change verified mode expiration.
+        mode = CourseMode.objects.get(course_id=self.course.id, mode_slug=CourseMode.VERIFIED)
+        mode.expiration_datetime = datetime.datetime(year=1970, month=1, day=1, tzinfo=UTC)
+        mode.save()
+
+        # It will not return the verified mode.
+        result_course = data.get_course_enrollment_info(unicode(self.course.id), include_expired=False)
+        result_slugs = [mode['slug'] for mode in result_course['course_modes']]
+        for course_mode in ['audit', 'honor']:
+            self.assertIn(course_mode, result_slugs)
```

Comment:
I don't see where that change is done. `test_get_course_with_expired_mode_included` and `test_get_course_without_expired_mode_included` share most of their lines.